app/viewModels/P5/departure/DepartureStatusP5ViewModel.scala

```diff
@@ -53,6 +56,16 @@ object DepartureStatusP5ViewModel {
       rejectedByOfficeOfDeparture(departureId, messageId, rejectionType, isDeclarationAmendable, xPaths)
     ).reduce(_ orElse _)
 
+  private def prelodgeRejectedStatus(
+    departureId: String,
+    messageId: String,
+    rejectionType: BusinessRejectionType,
```

Comment:
Again `rejectionType` is pretty obsolete here. If it's a prelodgeRejectedStatus then we known it's a PrelodgeRejectedMovementAndMessage